.github/workflows/e2e.yaml

```diff
@@ -60,6 +60,7 @@ jobs:
           - "install_merlin" install_merlin \
           - "build_turing_router_docker_image" build_turing_router_docker_image \
           - "build_turing_apiserver_docker_image" build_turing_apiserver_docker_image \
+          - "build_turing_swagger_docker_image" build_turing_swagger_docker_image \
```

Comment:
Do we need it here? Can we for e2e test just use `swagger/swagger-ui:latest` or smth?

Reply:
It doesn't take long anyway but we should follow the full flow as much as possible.